posts/fandl.md

```diff
@@ -0,0 +1,191 @@
+---
+title_word: FANDL
+title: FANDL Database View
+tags: ['post', 'fandl', 'sql', 'ssms']
+authors: ['Brian']
+date: 2019-10-23
+summary: "An overview of how to connect and write queries against the FANDL Warehouse."
+---
+
+## Overview
+
+The following will lay out how to:
+
+* Connect to the FANDL Database
```

Comment:
Did not know about this tag.
Since I'll be removing all instances of FANDL from the document, this change won't be applied, but I'll keep an eye out for other instances where it may come in handy.